makefile

```
@@ -0,0 +1,60 @@
+.PHONY: edit build rebuild run logs logs-follow attach stop test clean release
+
+IMAGE_LABEL := bw-local-dev
+VOLUME_LABEL := $(IMAGE_LABEL)-volume
+CONTAINER_ID := $(shell docker ps --filter ancestor=$(IMAGE_LABEL) --format '{{.Names}}')
+RELEASE_DIR := $(HOME)/btt-writer-release
+
+edit:
+	# Edits common files in your favorite editor
+	$(EDITOR) makefile Dockerfile src/js/*.js src/views/*.html src/elements/**/*.html src/css/*.css
+
+build:
+	# Builds image
+	docker build --tag bw-local-dev .
+
+rebuild:
+	# Re-builds image from scratch (takes a while!)
+	docker build --no-cache --tag bw-local-dev .
+
+run:
+	# Starts BTT-Writer in a container.  The user directory is in a volume.
+	# Runs detached to prevent messing up the tty.  (Use `make stop` to end if needed.)
+	test $(DISPLAY) # If blank, then $$DISPLAY is not set
+	test ! $(CONTAINER_ID) # If not blank, then container is already running
+	docker run --detach --rm --volume $(VOLUME_LABEL):/root --env DISPLAY="${DISPLAY}" $(IMAGE_LABEL)
```

I needed this to get it to run on native linux:
```suggestion
	docker run --detach --rm --net=host --volume="${HOME}/.Xauthority:/root/.Xauthority:rw" --volume $(VOLUME_LABEL):/root --env DISPLAY="${DISPLAY}" $(IMAGE_LABEL)
```

---

OK, cool -- I'll check and see if that breaks WSL...

---

Seems to work fine in WSL.